qa-include/plugins/qa-widget-related-qs.php

```diff
@@ -65,8 +65,12 @@ public function output_widget($region, $place, $themeobject, $template, $request
 			$themeobject->output('<ul class="qa-related-q-list">');
 
 			foreach ($questions as $question) {
+				$isASelected = $question['selchildid'] != null ? ' qa-a-count-selected' : '';
+				$qaLangAnswers = str_replace(':', '', qa_lang_html('profile/answers'));
```

Comment:
I wouldn't manipulate or make any assumption on user input. Languages are very different from each other. I'd suggest to use this instead: `main/x_answers`